packages/store-ui/src/Minicart/Drawer/Header/CloseButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Button } from 'theme-ui'
+import type { FC } from 'react'
+
+interface Props {
+  variant: string
+  onClose?: () => void
```

Comment:
I think it's better to change to `React.MouseEventHandler<HTMLButtonElement> | undefined`, or it will trigger an error if you want to get the event param.